_config/features_comparison.yml

```diff
@@ -0,0 +1,241 @@
+features:
+  - name: "ACID"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
+    blacktie: true
+    arjuna: true
+    txoj: true
```

Comment:
I think we should go with those additional observations then. What I wanted was for it to be clear what properties are provided from where. So lets tick ACID for TXoJ and ArjunaCore but call out these specific observations below the table.